gimme

```diff
@@ -286,6 +287,20 @@ _wipe_version() {
 	fi
 }
 
+_list_versions() {
+	(
+		if [ ! -d "${GIMME_VERSION_PREFIX}" ] ; then
+			return 0
+		fi
+
+		cd "${GIMME_VERSION_PREFIX}"
+		for d in $(ls -1d *) ; do
```

Comment:
```bash
_list_versions() {
	if [ ! -d "${GIMME_VERSION_PREFIX}" ] ; then
		return 0
	fi

	for d in "${GIMME_VERSION_PREFIX}/go"*".${GIMME_OS}." ; do
		local lessgo="${d##*/go}"
		echo "${lessgo%%.${GIMME_OS}.*}"
	done
}
```
(totally untested of course, just because)